definitions.go

```diff
@@ -675,10 +675,28 @@ type UserFederationProviderRepresentation struct {
 	ProviderName      *string                 `json:"providerName,omitempty"`
 }
 
+// AttributeKey type
+type AttributeKey string
+
+// Attribute keys definition
+const (
+	AttrbBirthDate           = "birthDate"
```

Comment:
In my opinion, it should not be part of keycloak-client, it is our way of doing it not linked to keycloak API